docs/5.0/pages/enterprise/quickstart-enterprise.md

```diff
@@ -39,11 +41,11 @@ the following DNS names and IPs:
 
 Server Name    |  IP Address    | Purpose
 ---------------|----------------|--------------
-_"auth.example.com"_  | 10.1.1.10      | This server will be used to run all three Teleport services: auth, proxy and node.
-_"node.example.com"_  | 10.1.1.11      | This server will only run the SSH service. The vast majority of servers in production will be nodes.
+_"auth.example.com"_  | 10.1.1.10      | This server will be used to run all three Teleport services: auth, proxy and ssh.
```

Comment:
The Teleport service is called `node`. I would suggest to leave it as `node` here for consistency. We could also consider `ssh node`.